src/mbed_tools/project/project.py

```diff
@@ -33,6 +33,8 @@ def clone_project(url: str, dst_path: Any = None, recursive: bool = False) -> No
         libs = LibraryReferences(root=dst_path, ignore_paths=["mbed-os"])
```

Comment:
We now return something, so the docstring should have some `Returns:` documentation.

Reply:
Documented.